src/routes/layout.stories.ts

```diff
@@ -10,4 +12,62 @@ export default {
   decorators: [() => LayoutDecorator]
 };
 
-export const Default = {};
+export const Default = {
+  play: async ({ canvasElement, step }) => {
+    const canvas = within(canvasElement);
+
+    await step('Copy email and check notification', async () => {
+      await userEvent.click(await canvas.findByTestId('email-icon'));
+      const notification = canvas.queryByText('O email foi copiado para o teu clipboard :)');
+      await expect(notification).toBeTruthy();
```

Comment:
just a suggestion and maybe this comes to personal preference, make a this string a global const variable, to make changes easier in the future, since you use it multiple times. Other thing I don't know if multiple languages are to be supported in the future and how this will affect tests.

Reply:
I agree. With translations, we'd likely have a map with different languages instead of constants. Imo that's a moderate refactor for the future